src/portable/portable.c

```diff
@@ -239,6 +242,7 @@ static int extract_now(
         _cleanup_(lookup_paths_free) LookupPaths paths = {};
         _cleanup_close_ int os_release_fd = -1;
         _cleanup_free_ char *os_release_path = NULL;
+        const char *os_release_file_path = "/etc/os-release";
```

Comment:
this is kinda confusing, i.e. where the difference between os_release_path and os_release_file_path is.

maybe rename the latter os_release_id? or so? since it's not really a path, we never open it as such, we just pass this as identifier to the parent